test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -1354,6 +1354,68 @@ TEST(LiteInterpreterTest, DefaultArgsPinvSpecifyDefault) {
   testLiteModuleCompareResultTensors(m, inputs);
 }
 
+void testDefaultArgsPinvWithOutArg(int num_args) {
+  Module m("m");
+  if (num_args == 1) {
+    m.define(R"(
+      def forward(self, input):
+        return torch.linalg_pinv(input, out=input)
+    )");
+  } else if (num_args == 2) {
+    m.define(R"(
+      def forward(self, input):
+        return torch.linalg_pinv(input, 1e-5, out=input)
+    )");
+  } else if (num_args == 3) {
+    m.define(R"(
+      def forward(self, input):
+        return torch.linalg_pinv(input, 1e-5, True, out=input)
+    )");
+  }
+
+  const int N = 28;
+  auto input = torch::range(1, N * N, 1);
+  input[0] = 1; // a more stable matrix
```

Comment:
input[0] is already 1. This line does not change input. It does not make the condition better. It's still a very ill-conditioned matrix. Maybe change input[0] = 10000 or something.

Reply:
Hmm can you elaborate it a bit more? Do you change `input[0]` to some other number?

Reply:
```
import torch
N = 3
input = torch.range(1, N * N, 1)
print(input)
```
```
tensor([1., 2., 3., 4., 5., 6., 7., 8., 9.])
```
`input[0] = 1` does not change input and the condition number of this matrix is Inf (very unstable) and the solution could vary in different platforms.